articles/azure-monitor/app/opencensus-python-request.md

```diff
@@ -113,6 +113,48 @@ First, instrument your Python application with latest [OpenCensus Python SDK](./
     config = Configurator(settings=settings)
     ```
 
+## Tracking FastAPI applications
+
+1. The following dependencies are required: 
+    - fastapi
+    - uvicorn
+
+2. Add FastAPI middleware.
+
+3. Set span kind server, `span.span_kind = SpanKind.SERVER`.
```

Comment:
This isn't really a step, as it is already done in step 2. I think for this step, we should say something like "Run your application. Calls made to your FastAPI application should be automatically tracked and telemetry sent directly to Azure Monitor..."

Reply:
Thank you, updated. let me know if further change is required.